### PR TITLE
48144851: Edge Privacy Whitepaper: Geolocation: Fix statement about InPrivate (doc bug)

### PR DESCRIPTION
**Pull request by vwehren**
Rendered article section for review:
https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-2972#geolocation -- end of section
Before:
https://learn.microsoft.com/microsoft-edge/privacy-whitepaper/#geolocation -- end of section

Edge privacy whitepaper states that InPrivate inherits the site permission for location from the originating profile from which in private was opened. This is incorrect. It is "Ask always", just like Guest Mode. This task addresses this bug by updating the section in the doc.

AB#48144939

### CODE REVIEW

**Review by mikehoffms — APPROVED** [review]
PR Approved per Writer/Editor review

File: microsoft-edge/privacy-whitepaper/index.md [hunk 1]
```diff
@@ -403,7 +403,7 @@ When the site you are on has access to your location, Microsoft Edge shows a loc
 
 Microsoft Edge doesn't collect or store the location data that is shared with the site, and generates a new random identifier for each request.
 
-**InPrivate** browsing uses the location permission settings of the browser profile from which the **InPrivate** session was launched. **Guest** mode always asks you for permission before granting a site access to your precise location.
+When you browse the web **InPrivate**, **Ask** and **Block** permissions to access location are inherited from the profile you used to open the **InPrivate** window. However, **Allow** is not automatically inherited and Microsoft Edge asks for your permission again before granting the site access to your precise location. In **Guest** mode, Microsoft Edge always asks for permission before granting a site access to your precise location.
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:406]: are there 3 permissions: Ask, Block, Allow?

[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:406]: alt, using ", which ," phrase:
When you browse the web using **InPrivate**, the **Ask** and **Block** permissions, which control accessing your precise location, are inherited from the profile that you used to open the **InPrivate** window.  However, the **Allow** permissions are not automatically inherited, and Microsoft Edge asks for your permission again before granting the site access to your precise location.  In **Guest** mode, Microsoft Edge always asks for permission before granting a site access to your precise location.

Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/index.md:406]: Could you reword these two sentences to make this more clear and easily readable? What are ask, block, and allow? 

`When you browse the web **InPrivate**, **Ask** and **Block** permissions to access location are inherited from the profile you used to open the **InPrivate** window. However, **Allow** is not automatically inherited and Microsoft Edge asks for your permission again before granting the site access to your precise location.`

**Review by apryzgoda — CHANGES_REQUESTED** [review]

File: microsoft-edge/privacy-whitepaper/index.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:406]: When you browse the web using **InPrivate**, the **Ask** and **Block** permissions that control accessing your precise location are inherited from the profile that you used to open the **InPrivate** window. However, the **Allow** permissions are not automatically inherited, and Microsoft Edge asks for your permission again before granting the site access to your precise location. In **Guest** mode, Microsoft Edge always asks for permission before granting a site access to your precise location.

[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:406]: used the "which" version

**Review by apryzgoda — APPROVED** [review]
Edge Privacy Champ Approved

**Review by mikehoffms — APPROVED** [review]
Writer/Editor approved PR

**Pull request merged**